### DISCUSSION TITLE
Adding Node and/or Express

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/reactsocialnetwork#22] [Ideas]
Would it be a good idea to add Express and Node?

**Comment by TechLateef** [comment]
Yh it will be it’s cool to work with node /express

**Reply by gbowne1** [reply]
yeah I figured.  I don't do that much with it.

My other project needs fixed though: https://github.com/gbowne1/radio_logger as it no longer starts and I have not been able to figure out what went wrong.   See also: I left a bunch of issues there too

**Reply by gbowne1** [reply]
I can get RL to connect to mongo.. but there's a whole pile of junk in the repo.

**Reply by TechLateef** [reply]
Alright let me check it out

**Reply by gbowne1** [reply]
React Social Media might need an API as well.

Another TODO item 👍😊

**Reply by TechLateef** [reply]
yeah sure
